### PR TITLE
A note about Homeworks and RadioRA:

### PR DESCRIPTION
**Pull request by burke**
I had to do some digging to figure out whether I'd be able to use this with Homeworks QSX. Here's a few helpful breadcrumbs for the next person in my situation.

### CODE REVIEW

**Comment by cbw** [comment]
Thanks for the doc updates! Besides the fact that leap-scan won't find a QSX processor (it advertises different mDNS than Caseta who-knows-why), any other issues using it with QSX?

File: pylutron_caseta/cli.py [hunk 1]
```diff
@@ -114,19 +114,22 @@ async def lap_pair(address: str, cacert: TextIO, cert: TextIO, key: TextIO):
...
+    $XDG_CONFIG_HOME/pylutron_caseta, named after the address of the bridge or
+    processor. The leap tool will look for certificates in the same location.
     """
 
     def _ready():
         click.echo(
-            "Press the small black button on the back of the bridge to complete "
-            "pairing."
+            "Press the pairing button on the bridge or processor to complete pairing. "
+            "Generally this is a small black button on the back of the device. "
```
Comment (mdonoughe) [inline pylutron_caseta/cli.py:130]: For Caseta it's a small black button on the back of the bridge. For QSX it's a small tan button on the front of the processor. For RA3 apparently it's a small black button labeled "program" on the front of the processor.

File: pylutron_caseta/cli.py [hunk 2]
```diff
@@ -114,19 +114,22 @@ async def lap_pair(address: str, cacert: TextIO, cert: TextIO, key: TextIO):
     """
     Perform LAP pairing.
 
-    This program connects to a Lutron bridge device and initiates the LAP pairing
-    process. The user will be prompted to press a physical button on the bridge, and
-    certificate files will be generated on the local computer.
+    This program connects to a Lutron bridge device or Homeworks processor and
```
Comment (mdonoughe) [inline pylutron_caseta/cli.py:117]: AFAIK this also applies to RadioRA3 processors.

File: README.md [hunk 3]
```diff
@@ -16,14 +16,22 @@ In order to communicate with the bridge device, you must complete the pairing pr
 
 If pylutron_caseta is installed with the cli extra (`pip install pylutron_caseta[cli]`), the `lap-pair` tool can be used to generate the certificate files. Simply running `lap-pair <BRIDGE HOST>` (note the LEAP port number should not be included) will begin the pairing process. The certificate files will be saved in `$XDG_CONFIG_HOME/pylutron_caseta` (normally `~/.config/pylutron_caseta`) in the files `[BRIDGE HOST]-bridge.crt`, `[BRIDGE HOST].crt`,  `[BRIDGE HOST].key`. Check `lap-pair --help` if you want to use different files.
 
+### A note on RadioRA and Homeworks
+
+This API primarily supports Lutron Caséta, but does mostly work with other Lutron products implementing the LEAP protocol, such as, RadioRA 3, and Homeworks QSX. Documentation here generally refers to Caséta and bridges, but most of it can be freely interpreted as relevant with "bridge" swapped out for "processor". A few key differences:
+
+* Homeworks QSX processors, at least, may not show up in `leap-scan`.
```
[the remark this answers is not in the document]
Comment (danaues) [reply README.md:23]: Confirmed.  leap-scan now discovers my RA3 Processor

**Pull request closed without merging**